### PR TITLE
allow cloud_admin to publicize images

### PR DESCRIPTION
**Pull request by zfeldstein**

### CODE REVIEW

**Comment by zfeldstein** [comment]
retest

**Comment by j2sol** [comment]
retest

File: roles/glance/templates/etc/glance/policy.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "get_image": "",
     "get_images": "",
     "modify_image": "",
-    "publicize_image": "role:admin",
+    "publicize_image": ["role:admin", "role:cloud_admin"],
```
Comment (lihkin213) [inline roles/glance/templates/etc/glance/policy.json]: this should be 
"role:admin or role:cloud_admin" using the new format for policy files.

Comment (j2sol) [inline roles/glance/templates/etc/glance/policy.json]: Ah, I went poking around to find docs on the format. I failed. Can you link to those?

**Comment by j2sol** [comment]
:+1:

**Comment by zfeldstein** [comment]
retest

**Comment by j2sol** [comment]
Lets backport this one to RELEASE-2.0.x please @zfeldstein

**Pull request merged**